Mod to work with redirected documents folders

problex opened the pull request:
Modified the script to use _([environment]::getfolderpath('mydocuments'))_ instead of _$ENV:UserProfile\Documents_ which will get the documents path even if it has been relocated to a network share or a different drive.

two renames to save enough characters for the longer command

sebkinne commented:
Hi @problex,

As @hak5darren has completely re-written this payload, I'm going to go ahead and close the PR.

problex commented:
OK should I try to update the code in the new version?
as even the new version does not work with computers using redirected
document folders.
I'm pretty sure this is why people are commenting saying their loot folder
is coming back empty.

On Fri, Apr 7, 2017 at 5:20 PM, Sebastian Kinne <notifications@github.com>
wrote:

> Closed #134 <https://github.com/hak5/bashbunny-payloads/pull/134>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hak5/bashbunny-payloads/pull/134#event-1033517194>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATX0yZEwPaX6B-fWAnDES4Jn-b1mjt9dks5rtce4gaJpZM4MvGr9>
> .
>

hak5darren commented:
I removed the old version since it had too many flaws fixed by faster_smb_exfiltrator and renamed that to simply smb_exfiltrator. 

The new version does the copy from s.ps1 - however this requires an execution bypass. 

In the event that execution bypass is unavailable, I'd like to offer the configurable option to use this robocopy directly from the run line technique.

Your PR including the mydocuments environment variable is much appreciated. I'll be looking at more PRs tonight/tomorrow so if you make this change to that payload I'll happily approve it, otherwise I'll add it and include your username in the credits & commit message. Thanks!

problex commented:
Awesome, yea would be choice if you could just put it into the code.
Works on AD Domain computers using folder redirection too, too bad
Bitdefender Gravityzone endpoint has a permission request pop up when
powershell is run.

Loved your last video BTW, I'm flying from NZ to the Gold Coast on Sunday,
wish I had the chance to catch up with the Hak5 team.

On Fri, Apr 7, 2017 at 7:48 PM, Darren Kitchen <notifications@github.com>
wrote:

> I removed the old version since it had too many flaws fixed by
> faster_smb_exfiltrator and renamed that to simply smb_exfiltrator.
>
> The new version does the copy from s.ps1 - however this requires an
> execution bypass.
>
> In the event that execution bypass is unavailable, I'd like to offer the
> configurable option to use this robocopy directly from the run line
> technique.
>
> Your PR including the mydocuments environment variable is much
> appreciated. I'll be looking at more PRs tonight/tomorrow so if you make
> this change to that payload I'll happily approve it, otherwise I'll add it
> and include your username in the credits & commit message. Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hak5/bashbunny-payloads/pull/134#issuecomment-292466808>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATX0yQHhkIajuS-Oc42kyr48W96cCfTMks5rtepggaJpZM4MvGr9>
> .
>
